bamboo/unit_tests/test_unit_check_proto_models.py

```diff
@@ -66,6 +66,7 @@ def skeleton_models(cluster, dir_name, executables, compiler_name,
                         time_limit = 3
                     if 'resnet50' in file_name:
                         node_count = 8
+                        continue
```

Comment:
If you want to skip `resnet50` probably best to add comment of why.

Reply:
I have updated this to just skip on non-`weekly` testing and I have added a comment to that effect.